[2024-02-24T20:42:44.244+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.Test_conexion scheduled__2023-12-18T00:00:00+00:00 [queued]>
[2024-02-24T20:42:44.312+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.Test_conexion scheduled__2023-12-18T00:00:00+00:00 [queued]>
[2024-02-24T20:42:44.315+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-24T20:42:44.462+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): Test_conexion> on 2023-12-18 00:00:00+00:00
[2024-02-24T20:42:44.490+0000] {standard_task_runner.py:60} INFO - Started process 633 to run task
[2024-02-24T20:42:44.503+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'Test_conexion', 'scheduled__2023-12-18T00:00:00+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp2qh5r0vp']
[2024-02-24T20:42:44.523+0000] {standard_task_runner.py:88} INFO - Job 328: Subtask Test_conexion
[2024-02-24T20:42:45.029+0000] {task_command.py:423} INFO - Running <TaskInstance: first_dag.Test_conexion scheduled__2023-12-18T00:00:00+00:00 [running]> on host ce9b59703482
[2024-02-24T20:42:45.545+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='Test_conexion' AIRFLOW_CTX_EXECUTION_DATE='2023-12-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-18T00:00:00+00:00'
[2024-02-24T20:42:45.584+0000] {sql.py:276} INFO - Executing: SELECT 1
[2024-02-24T20:42:45.634+0000] {base.py:83} INFO - Using connection ID 'postgres1' for task execution.
[2024-02-24T20:42:45.687+0000] {base.py:83} INFO - Using connection ID 'postgres1' for task execution.
[2024-02-24T20:42:45.747+0000] {sql.py:450} INFO - Running statement: SELECT 1, parameters: None
[2024-02-24T20:42:45.752+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-24T20:42:45.847+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=Test_conexion, execution_date=20231218T000000, start_date=20240224T204244, end_date=20240224T204245
[2024-02-24T20:42:45.912+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-24T20:42:45.984+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
